pkg/limayaml/defaults.go

```diff
@@ -630,6 +637,25 @@ func NewArch(arch string) Arch {
 	}
 }
 
+func NewVMType(driver string) VMType {
+	switch driver {
+	case "vz":
+		return VZ
+	case "qemu":
+		return QEMU
+	default:
+		logrus.Warnf("Unknown driver: %s", driver)
+		return driver
+	}
+}
+
+func ResolveVMType(s *string) VMType {
+	if s == nil || *s == "" || *s == "default" {
+		return NewVMType("qemu")
+	}
+	return *s
```

Comment:
Nitpicking (you only need to call `NewVMType` when you potentially have an unknown driver name):

```suggestion
	if s == nil || *s == "" || *s == "default" {
		return QEMU
	}
	return NewVMType(*s)
```

Reply:
Done